User data

laurenswart opened the pull request:
L'utilisateur authentifié peut voir ses données perso et ses réservations, et peut également modifier son nom, prenom, email, login, langue
Il peut aussi réinitialiser son mot de passe. Attention à configurer le fichier .env par rapport à votre compte mailtrap

laurenswart committed:
affichage des données personnelles

laurenswart committed:
afficher le formulaire de modif données perso

laurenswart committed:
sauver modifications des données perso

laurenswart committed:
Réinitialisation de mot de passe

cedryc-ruth commented on resources/views/user/account.blade.php:
```diff
@@ -0,0 +1,73 @@
+                            <td><a href="{{ route('locations_show', $reservation->representation->location_id) }}">{{ $reservation->representation->location->designation }}</a></td>
+                            <td><a href="{{ route('shows_show', $reservation->representation->show_id) }}">{{ $reservation->representation->show->title }}</a></td>
+                            <td>{{ $reservation->places }}</td>
+                            <td>{{ $reservation->places * $reservation->representation->show->price }} &#8364</td>
```
&#8364;

Terminer par un poitn-virgule

cedryc-ruth commented on resources/views/user/account.blade.php:
```diff
@@ -0,0 +1,73 @@
+                    </tbody>
+                </table>
+            @else 
+                <p>Aucune réservation à venir.
```
Fermer le p
</p>

Zangetsu95 commented:
si après les commentaires du prof ,normalement tout est ok :) !

laurenswart committed:
bugs fixed

registration/login form use old()
account change data: backend email validation

laurenswart commented:
Voilà j'ai réglé les erreurs de syntaxe mentionnés au cours, les formulaires de login/registration utilisent old(), et la validation du changement d'email comprend une validation backend que c'est bien un email.
